author.tf

```diff
@@ -514,12 +514,12 @@ module "survey-register" {
   vpc_id                 = "${module.author-vpc.vpc_id}"
   aws_alb_arn            = "${module.author-eq-ecs.aws_external_alb_arn}"
   aws_alb_listener_arn   = "${module.author-eq-ecs.aws_external_alb_listener_arn}"
-  service_name           = "survey-register"
+  service_name           = "author-survey-register"
```

Comment:
Is this change still required now that survey register has been removed from eq-terraform?

Reply:
Not anymore, just thought I might as well make it consistent.